SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitApi.java

```diff
@@ -46,11 +46,53 @@ public void onResult(String responseBody, Exception httpError) {
         }
     }
 
-    void tokenize(String ibanLastFour, String customerId, String bankReferenceToken, String mandateType) {
-        // TODO: implement (future PR)
+    void tokenize(String ibanLastFour, String customerId, String bankReferenceToken, String mandateType, final SEPADirectDebitTokenizeCallback callback) {
+        try {
+            HttpRequest httpRequest = buildTokenizeHttpRequest(ibanLastFour, customerId, bankReferenceToken, mandateType);
+            httpClient.sendRequest(httpRequest, new HttpResponseCallback() {
+                @Override
+                public void onResult(String responseBody, Exception httpError) {
+                   if (responseBody != null) {
+                       try {
+                           SEPADirectDebitNonce nonce = parseTokenizeResponse(responseBody);
+                           callback.onResult(nonce, null);
+                       } catch (JSONException jsonException) {
+                           callback.onResult(null, jsonException);
+                       }
+                   } else if (httpError != null) {
+                       callback.onResult(null, httpError);
+                   }
+                }
+            });
+        } catch (JSONException e) {
+            callback.onResult(null, e);
+        }
+    }
+
+    private SEPADirectDebitNonce parseTokenizeResponse(String responseBody) throws JSONException {
+        JSONObject jsonResponse = new JSONObject(responseBody);
+        return SEPADirectDebitNonce.fromJSON(jsonResponse);
+    }
+
+    private HttpRequest buildTokenizeHttpRequest(String ibanLastFour, String customerId, String bankReferenceToken, String mandateType) throws JSONException {
```

Comment:
This logic will change once the feature is in Sandbox - we are hard coding some request params while testing locally